sysutils/ansible-lint/Portfile

```diff
@@ -4,26 +4,40 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        willthames ansible-lint 3.4.19 v
+github.setup        willthames ansible-lint 5.0.7 v
 name                ansible-lint
 categories          sysutils
 platforms           darwin
 supported_archs     noarch
 license             MIT
-maintainers         artisancomputer.com:zdw \
+maintainers         artisancomputer.com:zdw outlook.de:judaew \
                     openmaintainer
 
 description         Best practices checker for Ansible playbooks
 long_description \
     ansible-lint checks playbooks for practices and behaviour that could potentially be improved
 
-checksums           rmd160  41da918b12b7d3ef9ae7cbee6e07ec653458771b \
-                    sha256  5ef527543ec8a6060da93aa4efcdeb360564242495d0a6e15fcbde071fa5decc
+checksums           rmd160  24ca7e4b1f9417ce9897aabe7d6ed0d9a30f5195 \
+                    sha256  6dc4e592005b7648b47b783fdfcc8f38af6d07c11bc6b13da5316616db198f24 \
+                    size    326613
 
-depends_run-append  path:bin/ansible:py27-ansible
-
-python.versions             27
-python.default_version      27
+python.versions             39
+python.default_version      39
 python.link_binaries_suffix
 
+depends_run-append  path:bin/ansible:py${python.version}-ansible
```

Comment:
why is this a `path` style dependency and not just the usual `port`?

Reply:
Any version of py-ansible is suitable for the port. I think `path` must allow a user to choose his target version of the py-ansible, even if it is different from asnible-lint.

Reply:
but that requires the end-user to do ` port select --list ansible` as stated in the notes. If you're using PY39 here wouldn't it make sense to use that version for `ansible` as well and (if needed) patch `ansible-lint` such that it works with `ansible-3.9` and such? That would avoid the need for doing `port select` and potentially make this more user-friendly for the enduser, but perhaps there is a reason that this isn't a good suggestion - I have no idea about the software.

Reply:
I have done a little research with different python versions of py-ansible. For ansible-lint, the python interpreter version must match the py-ansible interpreter version.

At the same time, the user will still need to execute `port select` after installation, since ansible-cli, in addition to importing, also runs the `ansible` in the shell as a subprocess.

https://github.com/ansible-community/ansible-lint/blob/1dc3533661c90c83cbf042880e2bd95b03d132c4/src/ansiblelint/config.py#L110-L150

I tried to fix it by adding python.prefix to `match` and `subprocess`, but that broke something elsewhere. I think `port select` is the best thing to do. In this case, everything works correctly.